DittoToolsApp/DittoToolsApp/AppSettings.swift

```diff
@@ -5,6 +5,30 @@
 import UIKit
 import DittoSwift
 
+extension AppSettings {
+    enum LogLevel:Int, CustomStringConvertible, CaseIterable {
+        case disabled = 0
+        case error, warning, info, debug, verbose
+        
+        var description: String {
+            switch self {
+            case .disabled:
+                return "disabled"
+            case .error:
+                return "error"
+            case .warning:
+                return "warning"
+            case .info:
+                return "info"
+            case .debug:
+                return "debug"
+            case .verbose:
```

Comment:
Can we remove verbose from the options? We don't want anyone using the `verbose` level so I'd rather hide it from end-users.